src/components/App/App.js

```diff
@@ -1,62 +1,36 @@
 import React, { lazy, Suspense } from 'react';
 import { IntlProvider } from 'react-intl';
 import { library } from '@fortawesome/fontawesome-svg-core';
-import { BrowserRouter, Route, Switch } from 'react-router-dom';
+import { BrowserRouter } from 'react-router-dom';
 import flatten from 'flat';
 
 import icons from 'assets/icons';
-import AppLocale from 'locales';
 import ErrorBoundary from 'components/ErrorBoundary';
-import Loading from 'components/Loading';
-import ProtectedRoute from 'components/ProtectedRoute';
+import FullPageSpinner from 'components/FullPageSpinner';
+import { useAuthentication } from 'hooks/auth';
 import { useLocale } from 'hooks/locale';
+import AppLocale from 'locales';
 
-import styles from './App.module.scss';
+import './App.module.scss';
 
-const ForgotPasswordPage = lazy(() => import('pages/ForgotPasswordPage'));
-const HomePage = lazy(() => import('pages/HomePage'));
-const NoMatchPage = lazy(() => import('pages/NoMatchPage'));
-const SettingsPage = lazy(() => import('pages/SettingsPage'));
-const SignInPage = lazy(() => import('pages/SignInPage'));
-const SignUpPage = lazy(() => import('pages/SignUpPage'));
+const AuthenticatedApp = lazy(() =>
+  import(/* webpackPrefetch: true */ '../AuthenticatedApp')
```

Comment:
is this comment needed? 🤔

Reply:
This will result in <link rel="prefetch" ...> being appended in the head of the page, which will instruct the browser to prefetch in idle time the AuthenticatedApp-chunk.js file.

https://webpack.js.org/guides/code-splitting/#prefetchingpreloading-modules